src/nanoplot/test.sh

```diff
@@ -0,0 +1,271 @@
+#!/bin/bash
+
+set -e
+
+## VIASH START
+meta_executable="$PWD/target/executable/nanoplot/nanoplot"
+meta_resources_dir="$PWD/src/nanoplot"
+## VIASH END
+
+###########################################################################
+
+# Test 1: Run NanoPlot with only input parameter
+
```

Comment:
But what is the point of creating temporary directories when by default the output will be saved in the default output directory in the config file?

Reply:
I don't think you need to create a separate TMPDIR per se, since the unit test will already be executed in a temp dir.

However I do agree that it would be nice if each separate test is run in its own directory, to make sure that you don't get any data leakage.

Example:

```bash
mkdir test1
pushd test1 # cd into directory
echo "> Run Test 1: one input (Fastq)"
# ... test code ...
popd # go back to original directory

mkdir test2
pushd test2
echo "> Run Test 2: multiple inputs (Fastq)"
# ... test code ...
popd
```